app/Entities/UserDelete.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Entities;
+
+use CodeIgniter\Entity\Entity;
+
+class UserDelete extends Entity
+{
+    protected $datamap = [];
+    protected $dates   = ['scheduled_at', 'created_at'];
+    protected $casts   = [
+        'user_id' => 'integer',
+    ];
+
+    /**
+     * Generate link to restore user account.
+     */
+    public function restore(): string
```

Comment:
```suggestion
    public function restoreLink(): string
```

My initial reaction with the function name was that this handled restoring the user, but since it just generates the link, can we name it a little more specifically?

Reply:
You're right, thanks!